recipe/meta.yaml

```
@@ -35,10 +34,10 @@ requirements:
     - pip
     - setuptools
     - wheel
-    - numpy {{ numpy }}
+    - numpy >=2.0,<2.3
```

```suggestion
    - numpy 2.0 # [py<313]
    - numpy 2.1 # [py==313]
```
This ensures the maximum compatibility (i.e. lowest lower bound).